lib/plan-builder-generated.js

```diff
@@ -1194,8 +1194,7 @@ pointLongitude(...args) {
     * @returns { CtsPolygon }
     */
 polygon(...args) {
-    // Adding types.XsString manually since cts.polygon now accepts string input (MarkLogic 11.1.0 or higher needed).
-    const paramdef = ['vertices', [types.CtsPoint, PlanColumn, PlanParam, types.XsString], false, true];
+    const paramdef = ['vertices', [types.CtsPoint, PlanColumn, PlanParam], false, true];
     const checkedArgs = bldrbase.makeSingleArgs('cts.polygon', 1, paramdef, args);
```

Comment:
Hi Aaron and Ling, the changes on lines 1197 and 1161 (cts.point and cts.polygon) came in as part of auto-generation but the tests are now failing. Please let me know if we can disregard the incoming changes from the auto-generation and revert it back.

Reply:
Should definitely revert those changes. This is a good example of a comment added to a generated file helping us understand why it should be reverted.